src/datepicker/types.js

```diff
@@ -49,6 +66,7 @@ export type WeekPropsT = {
   onDayMouseOver: ({date: Date, event: Event}) => void,
   onDayMouseLeave: ({date: Date, event: Event}) => void,
   onSelect: ({date: Date}) => void,
+  overrides?: DatepickerOverridesT<{}>,
```

Comment:
should `SharedStyleProps` be passed as an arg to the type?

Reply:
`SharedStyleProps` only passed to the `Day` component, so I didn't add it here